common/etc/nginx/templates/default.conf.template

```diff
@@ -77,6 +77,9 @@ server {
     location /aws/credentials/retrieve {
         internal;
         js_content s3gateway.fetchCredentials;
+        
+        # Enable the below line if you are seeing SSL Handshake error while connecing to s3 bucket
+        # js_fetch_trusted_certificate /etc/ssl/certs/Amazon_Root_CA_1.pem;
```

Comment:
Maybe it would be worth it to allow to enable this by ENV variable?

Cause based on the comment message it seems that this is a known issue. In case of this users would be forced to rebuild an image or mount a config with just a single line uncommented. It would not be so great UX, I believe. :)